tests/post/features/service_configuration.feature

```diff
@@ -9,3 +9,14 @@ Feature: Cluster and Services Configurations
         And we apply the 'metalk8s.addons.dex.deployed' salt state
         Then we have '3' at 'status.available_replicas' for 'dex' Deployment in namespace 'metalk8s-auth'
         And we restore original state of 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth'
+        And we apply the 'metalk8s.addons.dex.deployed' salt state
+
+    Scenario: Customization of pre-defined Prometheus rules
+        Given the Kubernetes API is available
+        And pods with label 'app=prometheus' are 'Ready'
+        And we have a 'metalk8s-prometheus-config' CSC in namespace 'metalk8s-monitoring'
+        When we update 'metalk8s-prometheus-config' CSC in namespace 'metalk8s-monitoring' 'spec.rules.node_exporter.node_filesystem_space_filling_up.warning.hours' to '48'
+        And we apply the 'metalk8s.addons.prometheus-operator.deployed.prometheus-rules' salt state
+        Then we have an alert rule 'NodeFilesystemSpaceFillingUp' in group 'node-exporter' with 'annotations.summary' equal to 'Filesystem is predicted to run out of space within the next 48 hours.'
+        And we restore original state of 'metalk8s-prometheus-config' CSC in namespace 'metalk8s-monitoring'
+        And we apply the 'metalk8s.addons.prometheus-operator.deployed.prometheus-rules' salt state
```

Comment:
Oh, sorry I didn't catch this before! It's really not a good way of doing cleanup: the `given` used to find a CSC should register teardown code for resetting to the default state. Same goes for the `then` used to trigger the update with Salt, I think. Let's keep it this way for now, as we really wish to get rid of BDD, but it's not a safe approach (if the first `then` assertion fails, you will not run other `then`s, hence will not run your teardown logic).